eng/source-build-patches/0001-global.json-updates-for-source-build.patch

```diff
@@ -0,0 +1,31 @@
+From d0c2069e7c81c5fbd5a27f786e5e3347ab9d4b95 Mon Sep 17 00:00:00 2001
+From: John Luo <johluo@microsoft.com>
+Date: Wed, 19 May 2021 10:41:41 -0700
+Subject: [PATCH] global.json updates for source build
+
+---
+ global.json | 5 +----
+ 1 file changed, 1 insertion(+), 4 deletions(-)
+
+diff --git a/global.json b/global.json
+index 5a2b8a613d..951a5f9fe3 100644
+--- a/global.json
++++ b/global.json
+@@ -5,13 +5,10 @@
+   "tools": {
+     "dotnet": "6.0.100-preview.5.21230.2",
+     "runtimes": {
+-      "dotnet/x64": [
++      "dotnet": [
+         "2.1.27",
+         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
+       ],
+-      "dotnet/x86": [
+-        "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
+-      ],
```

Comment:
1. Why do we need _any_ additional runtimes i.e. anything but `runtimes/dotnet/$(MicrosoftNETCoreBrowserDebugHostTransportVersion)` in this array❔ The 2.1, 3.1, and x86 bits should only be used in testing.
2. Can we please do this w/ a `jq` script❔ It's my understanding prebuilt _tools_ are fine when source-building.

Reply:
I'm not completely sure but maybe the runtime is used for our x86 builds? I wouldn't remove it from out global.json but I'll change it to a jq script or something.

Reply:
Looks like the container doesn't have jq. Maybe a repo task?

Reply:
> Looks like the container doesn't have jq

We install it ourselves. See https://github.com/dotnet/aspnetcore/blob/5de71af529a5cc3780cbab0d00b18d43472b4cce/.azure/pipelines/ci.yml#L766-L772

Reply:
Is that an option in source build? How open are we to downloading things from Github? I would imagine that's a prebuilt?

Reply:
I just wrote a pure bash script to handle the global json updates.